ui/main.qml

```diff
@@ -152,9 +152,6 @@ StatusWindow {
             else if(state === Constants.appState.main) {
                 // We set main module to the Global singleton once user is logged in and we move to the main app.
                 Global.mainModuleInst = mainModule
-                if (Qt.platform.os === Constants.windows) {
-                    windowsOsNotificationsConnection.enabled = true
-                }
```

Comment:
Maybe there was a reason to make `windowsOsNotificationsConnection` alive only after we logged in? Otherwise, I think it would not make sense to write `enabled: false` deliberately. @jrainville git blame shows it was introduced there: 2f338ecb09c4fbac4a617f124eef35261520d1a0, was there any issue with making it enabled before logging in?

Reply:
The reason was that I knew `Global.mainModuleInst` was undefined/null until we login. As long as it doesn't throw a warning and it still work, I'm fine with setting the condition above.